server/fleet/service.go

```diff
@@ -384,7 +384,7 @@ type Service interface {
 	// OSVersions returns a list of operating systems and associated host counts, which may be
 	// filtered using the following optional criteria: team id, platform, or name and version.
 	// Name cannot be used without version, and conversely, version cannot be used without name.
```

Comment:
name_only cannot be used without version and conversely, version cannot be used without name_only?

Reply:
correct, that's the existing API behavior, but we could add new params without breaking the API